test/pylib/manager_client.py

```diff
@@ -219,6 +219,20 @@ async def all_servers(self) -> list[ServerInfo]:
         return [ServerInfo(ServerNum(int(info[0])), IPAddress(info[1]), IPAddress(info[2]), info[3], info[4])
                 for info in server_info_list]
 
+    async def starting_servers(self) -> list[ServerInfo]:
+        """Get List of server info (id and IP address) of servers currently
+           starting. Can be useful for killing (with server_stop()) a server
+           which a test started in the background but now doesn't expect to
+           ever finish booting successfully.
+        """
+        try:
+            server_info_list = await self.client.get_json("/cluster/starting-servers")
+        except RuntimeError as exc:
+            raise Exception("Failed to get list of starting servers") from exc
+        assert isinstance(server_info_list, list), "starting_servers_ids got unknown data type"
+        return [ServerInfo(ServerNum(int(info[0])), IPAddress(info[1]), IPAddress(info[2]), info[3], info[4])
```

Comment:
Even with the fact that JSON is small, I think more readable will be automatically unpack this.
```python
return [ServerInfo(**json.loads(info)) for info in server_info_list]
```